crypt_test.go

```diff
@@ -36,3 +36,61 @@ func TestEncryptionMechanism(t *testing.T) {
 func TestHashing(t *testing.T) {
 	assert.Equal(t, hashing("unsupportHashAlgorithm", []byte{}), []uint8([]byte(nil)))
 }
+
+func TestGenISOPasswordHash(t *testing.T) {
+	/*
```

Comment:
I think this example can be removed, developer's can be determined fields would be like by data structure definition.